src/loop_cache.cpp

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
What is the purpose of this file?

Reply:
Just placeholder making sure the new version is good. Could move implementation into it if that is something we want, but otherwise I will delete.

Reply:
I'm always of the opinion that the implementation should live in the source file when possible.

Reply:
Since this is not templated, it's an easy move, so I will.